### ISSUE TITLE
halLiftover vs UCSC liftover

### ISSUE THREAD

**Issue by mglgnn** [ComparativeGenomicsToolkit/hal#219]
Hi all,
I want to project one annotation from a source to a destination genomes (from two possibly distant species). I am interested in the result of mapping for exons.

I ran halLiftover with success but I wonder whether a different approach is to be preferred.

I quote one of your online tutorials:
_Chaining outside of HAL is necessary, as the pairwise alignments extract from HAL are fragmented and lack long-range continuity._

So, should I create chains according to that protocol and then apply original UCSC liftover to avoid fragmentation or it is safe to use halLiftover directly? 

Thanks for the usual kind support,
Giovanna

**Comment by glennhickey** [comment]
Yes, I think that's correct.  @diekhans may have more information?  This is something that we've been meaning to overhaul for a while now, but just never find the time.
